src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -275,6 +275,21 @@ public static String getParquetFilePath(String targetTableName, String flightId)
     return "parquet/" + targetTableName + "/" + flightId + ".parquet";
   }
 
+  public static String getSnapshotParquetFilePath(UUID snapshotId, String targetTableName) {
+    return "parquet/" + snapshotId + "/" + targetTableName + ".parquet";
+  }
+
+  public static String getSourceDatasetParquetFilePath(String tableName, String datasetFlightId) {
+    if (datasetFlightId != null) {
```

Comment:
What's the reason for `datasetFlightId` being null?

Reply:
We want to query all of the parquet files ingested into a dataset when creating a byFullView snapshot, but this test uses a single dataset and tests running in parallel could get different results. So the test uses the `datasetFlightId` to query the parquet file created by the test. If the flight id is `null`, we query parquet files from every flight. Another option is to make the test create a new dataset each time, but that adds to how long the test takes.

Reply:
Ah, got it. Thanks!